src/EthMultiVault.sol

```diff
@@ -2178,15 +2364,15 @@ contract EthMultiVault is IEthMultiVault, Initializable, ReentrancyGuardUpgradea
             revert Errors.EthMultiVault_VaultNotTriple();
         }
 
-        return vaults[type(uint256).max - id].balanceOf[receiver] > 0;
+        return vaults[getCounterIdFromTriple(id)].balanceOf[receiver] > 0;
```

Comment:
3.16 - Utilize Helper Functions More

Here, we make better use of the helper methods as a single source of truth.